### ISSUE TITLE
Cannot check text - confirm privacy policy first on microsoft teams website

### ISSUE THREAD

**Issue by FederAndInk** [languagetool-org/languagetool#9587]
On the microsoft teams website (https://teams.microsoft.com/) I constantly need to click the red exclamation point to confirm the privacy policy

![Capture d’écran du 2023-11-02 15-53-51](https://github.com/languagetool-org/languagetool/assets/24270152/ac1620b1-a8eb-40a1-ae66-ca8ee65a3a20)

OS: Debian 12 Linux
Browser: Firefox 115.3.0esr
LanguageTool addon version: 7.1.13
with the Premium subscription

**Comment by tiff** [comment]
Are you using Firefox profiles?

**Comment by FederAndInk** [comment]
I am using firefox containers, not profiles though

**Comment by tiff** [comment]
Yes, sorry, that's what I meant. It's a known problem that we don't have a solution for currently.

**Comment by arathunku** [comment]
Same happens on Slack, additionally for me - it also breaks built-in browser's spellcheck and excluding the domain  doesn't help. I've to completely disable the extension.

If given website is excluded, it shouldn't load anything.
